popolo/importers/popit.py

```diff
@@ -374,8 +374,17 @@ def update_membership(
     ):
         Membership = self.get_popolo_model_class('Membership')
         if 'id' not in membership_data:
-            membership_data['id'] = hash(str(membership_data)) # TODO rethink this line!
-
+            #construct an 'id' based on data that should be unique_together
+            new_id = ''
```

Comment:
I think it'd be good to factor out this code into a `synthesize_membership_id` method (or some better name, that's just what occurred to me).